pom.xml

```diff
@@ -127,7 +127,7 @@
                 </configuration>
             </plugin>
 
-            <plugin>
+            <!--<plugin>
```

Comment:
I'm not very familiar with Java, but are you commenting out this block?  If so, mind just deleting if we no longer need it?

Reply:
removing comment mark - fount another way to skip javadoc generation

Reply:
done